FifthLesson/VectorProgram/VectorProgram/ProgramRun.cs

```diff
@@ -13,87 +9,125 @@ class ProgramRun
     {
         public void Run()
         {
-            Vector vectorFirst = EnterUserData("first");
-            Vector vectorSecond = EnterUserData("second");
+            var vectors = CreateVectors();
 
-            ShowVectorsAndActions(vectorFirst, vectorSecond);
-            ShowMultiplicationAndVectors(vectorFirst, vectorSecond);
-            CompareVectors(vectorFirst, vectorSecond);
+            ShowVectors(vectors[0], vectors[1]);
```

Comment:
I meant all these operations when I suggested to group them into one method

Reply:
As for me this method is too big. I think that you can group some method with the similar logic, e.g. for showing multiplication results.

Let's discuss it on monday